src/zeroconf/_listener.pxd

```diff
@@ -1,13 +1,13 @@
 
 import cython
 
+from ._protocol.incoming cimport DNSIncoming
+from ._utils.time cimport current_time_millis, millis_to_seconds
```

Comment:
We still have a lot of float conversions from the constants that we can't patch because it breaks the tests